packages/svelte/src/compiler/phases/2-analyze/index.js

```diff
@@ -431,6 +431,26 @@ export function analyze_component(root, source, options) {
 		template,
 		elements: [],
 		runes,
+		// if we are not in runes mode but we have no reserved references ($$props, $$restProps)
+		// and no `export let` we might be in a wannabe runes component that is using runes in an external
+		// module...we need to fallback to the runic behavior
+		maybe_runes:
```

Comment:
you should also check for `$:` statements

Reply:
Uh right...I assumed it was explicit but it's just JavaScript...let me add it